src/components/WalletSelector/multichain/MultichainWalletSelector.tsx

```diff
@@ -162,20 +163,10 @@ export function MultichainWalletSelector({
           <WalletButton label="Solana" icon="solana" disabled />
         </VStack>
       </VStack>
-    </StyledWalletSelector>
+    </WalletSelectorWrapper>
   );
 }
 
-const StyledWalletSelector = styled(VStack)`
-  text-align: center;
-  width: 320px;
-
-  @media only screen and ${breakpoints.mobileLarge} {
-    width: 400px;
-    max-width: 480px;
-  }
-`;
-
```

Comment:
factored this out since i needed it in the fallback ui and didn't want to import this file as that would defeat the purpose of the codesplitting